### DISCUSSION TITLE
✅ 🥳 不止中文，只要是包内的字符，统统分包

### DISCUSSION THREAD

**Discussion by wanghaisheng** [KonghaYao/cn-font-split#110] [Q&A] [answered]
这个包内的字符，包有格式限制吗  字符呢 有大神解释下不

**Comment by KonghaYao** [comment]
字体需要为 otf 或者是 ttf 字体，均为常见的字体形式。字符是直接读取字体内部的所有字符，然后进行处理的。由于我们使用的是业内常用的字体渲染工具 harfbuzz 进行子集化，所以常见的 feature 也都会被纳入分包的范围。
即将推出的 V7 版本在此之上大幅度提升了性能，并优化了 feature 及各种语言的处理，@wanghaisheng

**Reply by wanghaisheng** [reply]
字符就是所谓的glyh吧

**Reply by KonghaYao** [reply, marked as answer]
代码里面是操作与分配 unicode 字符，harfbuzz 会自动关联 glyph 并抽取出来。想要表达的意思是，不只中文范围内的 unicode 字符可以抽取，阿拉伯、藏文、壮语的字符都会纳入分包范围。

**Reply by wanghaisheng** [reply]
明白了 就是只要它是这个glyph 不管牛鬼蛇神， 我能不能直接用ai生图造glyph 丢给他打包成我自己的字体库

**Reply by KonghaYao** [reply]
cn-font-split 负责的是将字体分为小字体包，如果你要制作字体，你可能要 svgtofont https://wangchujiang.com/svgtofont/，或者你不是 svg 就要其它工具，一般来说数量少的话，不需要使用 cn-font-split。
